MOBILE_API.xml

```diff
@@ -3830,6 +3830,13 @@
                 Audiobook: The total number of sections in the playback queue (e.g. the number of chapters in the book)
             </description>
         </param>
```

Comment:
Does there need to be a line break here, like there is between other params?

Reply:
Updated